src/app/projects/create-project/create-project.styl

```diff
@@ -87,6 +87,24 @@ md-select.create-project-select
   text-align center
   line-height 40px
 
+.create-project-panel-progress-item-number-in-progress
+  animation: pulse 5s infinite;
+
+@keyframes pulse {
+  0% {
+    background-color white
```

Comment:
it's to write only the essential stuff ;-)
http://stylus-lang.com/docs/selectors.html